langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -110,6 +143,11 @@ public isolated function concat((xml|string)... xs) returns xml = @java:Method {
 
 # Returns a string giving the expanded name of an xml element.
 #
+# ```ballerina
+# xml:Element p = xml `<person age="30">John</person>`;
```

Comment:
Shall we change it as: 
```suggestion
# xml `<person age="30">John</person>`.getName() ⇒ person;
```
shall we also add an example for an invalid case

Reply:
The `.getName()` could only be applied on `xml:Element` type.
https://github.com/ballerina-platform/ballerina-lang/pull/39130/files#diff-12a3bef6fbe98910da31aff32a36c7e3d51e96c16ba1e7e8e6672f2835bf3abbR153

Therefore, it would not be suggested on other xml subtypes.